index.html

```diff
@@ -14,36 +15,2214 @@
 <body>
   <div class="container">
     <div class="header">
-      <h1>Week 1: HTML Tables</h1>
-      <p>We'll add our data to this table.</p>
+      <h1>Week 1: Unemployment Rates Since 1980</h1>
+      <p>This table shows two unemployment rates measured every month since 1980.
+	  The second column is the civilian unemployment rate. The third column is the percentage out of total unemployed
+	  who have been unemployed for 27 weeks or more.
+	  <br>
+	  <br>
+	  
```

Comment:
Looks like we're missing the close to the P tag. Also, I'd recommend avoiding the BR elements. We could achieve the same goal with styles like this: `.header { padding-bottom: 32px; }`

Maybe this should be:

```
<p>This table shows two unemployment rates measured every month since 1980.</p>
<p>The second column is the civilian unemployment rate. The third column is the percentage out of total unemployed who have been unemployed for 27 weeks or more.</p>
```